### ISSUE TITLE
Ansible-Lint Pre-Commit Hook - couldn't resolve module/action

### ISSUE THREAD

**Issue by Cust0mfirmware** [ansible/ansible-lint#4365]
##### Summary

If I try to commit something and the ansible-lint pre-commit hook is triggered, i am getting the following error message:

couldn't resolve module/action 'community.general.lxc_container'.

I have installed the community.general collection via ansible-galxy and if I execute ansible-lint manually, the module can be resolved.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint 6.17.2 using ansible-core:2.16.3 ansible-compat:4.1.11 ruamel-yaml:0.17.21 ruamel-yaml-clib:0.2.8
```

- ansible installation method: OS package

##### STEPS TO REPRODUCE

1. Make a commit
2. Pre-commit hook is triggered
3. Ansible-lint is executed automatically

##### Desired Behavior

Module community.general.lxc_container should be resolved.

##### Actual Behavior

Error: couldn't resolve module/action 'community.general.lxc_container'.

**Comment by Cust0mfirmware** [comment]
Fixed by adding the following to the pre-commit config file:

```
# Uncomment if you need the full Ansible community bundle instead of ansible-core:
  additional_dependencies:
    - ansible
```
Source: https://ansible.readthedocs.io/projects/lint/configuring/#pre-commit-setup
